docs/UserGuide.md

```diff
@@ -115,32 +116,51 @@ Displays the output of `View` command, which expands each ingredient or recipe c
   e.g. `add ingredient -n <name> -q <quantity>[<units>]` can be used as:
      1. `add ingredient -n chicken -q 2` or as
      2. `add ingredient -n chicken -q 2kg`
+    
+- Items in parentheses `()` should come together.
+
+    e.g. `edit ingredient <index> (-<field flag> <new data>)...` can be used as:
+     1. `edit ingredient 3 -q 3` or as 
+     2. `edit ingredient 3 -q 3 -e 11-03-2022`
 
 - Items with `…` after them can be used multiple times.
 
   e.g. `find ingredient <keyword>...`, can be used as:<br />`find ingredient Strawberry Milk Cheese Tomato`
 
-## 4.1 General Commands
+## 4.2 Command Flags
+
+ Flag | Usage | Description | Compulsory? | Remarks 
+---- | ------- | ----------- | ------- | ----
+`-n` | `-n <name>` | Name of the ingredient or recipe | Yes | Names can only contain alphanumeric characters or spaces.
+`-q` | `-q <quantity> [unit]` | Quantity of the ingredient | Yes | Must be a number. Can be followed by a unit of measurement (`g`, `kg`, `mg`, `l`, `ml`, `kl`). Quantities will be converted to grams or litres, and numbers rounded off to 3 decimal places.
+`-i` | `-i <name> <quantity>...` | Ingredients used in the recipe | At least one specified | Must be a name followed by a space and a quantity.
+`-e` | `-e <expiry date>` | Expiry date of ingredients | Yes | Must be in the form DD-MM-YYYY.
+`-s` | `-s [step]...` | Step used in the recipe | No | Can contain any characters or spaces. 
+`-d` | `-d [description]` | Description of the recipe or ingredient | No | Can contain any characters or spaces.
+`-t` | `-t [tag]...` | Tag for the ingredient | No | Can contain alphanumeric characters or spaces.
 
-### 4.1.1 Help
+## 4.3 General Commands
+
+### 4.3.1 Help
 **Format:**<br />
 `help`
 
 Pops out a window that leads the user to [User Guide](https://ay2122s1-cs2103t-w11-1.github.io/tp/UserGuide.html)
 (You are here).
 
-### 4.1.2 Exit
+### 4.3.2 Exit
 **Format:**<br />
 `exit`
 
 Closes the window and exits the program. All your information will be saved.
 
-## 4.2 Ingredients
-
-This sections covers commands related to Inventory management. Any command primarily interacting with ingredients
-will be here.
+## 4.4 Ingredients
+This section covers commands related to Inventory management. Any command primarily interacting with ingredients will 
+be here.
+- Note that by default, Fridgy will sort all Ingredients by expiry dates in descending order i.e. soonest expiring
+  item will be at the top.
 
-### 4.2.1 Add Ingredients
+### 4.4.1 Add Ingredients
```

Comment:
The command format is 

`add ingredient -n <name> -q <quantity>[<units>] [-d <description>] -e <expiry date> [-t <tags>]`

The standard approach is to put optional flag at the back like 

`add ingredient -n <name> -q <quantity>[<units>]  -e <expiry date> [-t <tags>] [-d <description>]`

Reply:
Also for examples, there are 3 examples but no explanation on what they are trying to show. For example, it should be like

1. All command flags used
command: `the command`

2. Minimal command flags used
command: `the command`